diff --git a/tests/examples/gl/gtk/fxtest/pixbufdrop.c b/tests/examples/gl/gtk/fxtest/pixbufdrop.c
index 3a43de2..5976e70 100644
--- a/tests/examples/gl/gtk/fxtest/pixbufdrop.c
+++ b/tests/examples/gl/gtk/fxtest/pixbufdrop.c
@@ -36,7 +36,7 @@
 #include <X11/Xlib.h>
 #endif
 
-static gint delay = 0;
+static gint test_delay = 0;
 static gint saveddelay = 0;
 static gint method = 1;
 
@@ -129,13 +129,13 @@ static gboolean
 set_location_delayed (gpointer data)
 {
   SourceData *sdata = (SourceData *) data;
-  delay--;
-  g_print ("%d\n", delay);
-  if (delay > 0) {
+  test_delay--;
+  g_print ("%d\n", test_delay);
+  if (test_delay > 0) {
     return TRUE;
   }
   g_object_set (G_OBJECT (sdata->data), sdata->nick, sdata->value, NULL);
-  delay = saveddelay;
+  test_delay = saveddelay;
   return FALSE;
 }
 
@@ -160,9 +160,9 @@ on_drag_data_received (GtkWidget * widget,
   userdata->nick = (gchar *) "location";
   userdata->value = g_strdup (filename);
   userdata->data = data;
-  saveddelay = delay;
-  if (delay > 0) {
-    g_print ("%d\n", delay);
+  saveddelay = test_delay;
+  if (test_delay > 0) {
+    g_print ("%d\n", test_delay);
     g_timeout_add_seconds (1, set_location_delayed, userdata);
   } else
     g_object_set (G_OBJECT (userdata->data), userdata->nick, userdata->value,
@@ -197,7 +197,7 @@ main (gint argc, gchar * argv[])
     {"method", 'm', 0, G_OPTION_ARG_INT, &method,
         "1 for gstdifferencematte, 2 for gloverlay", "M"}
     ,
-    {"delay", 'd', 0, G_OPTION_ARG_INT, &delay,
+    {"test_delay", 'd', 0, G_OPTION_ARG_INT, &test_delay,
           "Wait N seconds before to send the image to gstreamer (useful with differencematte)",
         "N"}
     ,
