diff --git a/ext/alsa/gstalsaplugin.c b/ext/alsa/gstalsaplugin.c
index e7894d2..d154877 100644
--- a/ext/alsa/gstalsaplugin.c
+++ b/ext/alsa/gstalsaplugin.c
@@ -37,7 +37,6 @@ plugin_init (GstPlugin * plugin)
 
   ret |= GST_ELEMENT_REGISTER (alsasrc, plugin);
   ret |= GST_ELEMENT_REGISTER (alsasink, plugin);
-  ret |= GST_ELEMENT_REGISTER (alsamidisrc, plugin);
 
   return ret;
 }
diff --git a/ext/alsa/meson.build b/ext/alsa/meson.build
index 025d2a9..e4ac6d0 100644
--- a/ext/alsa/meson.build
+++ b/ext/alsa/meson.build
@@ -1,7 +1,6 @@
 alsa_sources = [
   'gstalsa.c',
   'gstalsadeviceprovider.c',
-  'gstalsamidisrc.c',
   'gstalsaelement.c',
   'gstalsaplugin.c',
   'gstalsasink.c',
@@ -9,7 +8,6 @@ alsa_sources = [
 ]
 
 alsa_headers = [
-  'gstalsamidisrc.h',
   'gstalsa.h',
   'gstalsaelements.h',
   'gstalsasrc.h',
